hack/e2e_compare.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Compare 2 files generated by e2e_collect. This ensures elements are sorted by version
+
+type -f jq > /dev/null || {
+  echo Please install jq
+  exit 1
+}
+
+diff <(jq -S '.nodes |= sort_by(.version)' ${1}) <(jq -S '.nodes |= sort_by(.version)' ${1}) || exit 0
```

Comment:
Would you mind adding a comment to describe what the expected is for the `jq` script?

Reply:
Done